HelpMyStreetFE/HelpMyStreetFE/src/sass/components/h-ticker.scss

```diff
@@ -0,0 +1,39 @@
+﻿/* (A) FIXED WRAPPER */
+.hwrap {
+    overflow: hidden; /* HIDE SCROLL BAR */
+    background: #eee;
+}
+
+/* (B) MOVING TICKER WRAPPER */
+.hmove {
+    display: flex;
+}
+
+/* (C) ITEMS - INTO A LONG HORIZONTAL ROW */
+.hitem {
+    flex-shrink: 0;
+    width: 100%;
+    box-sizing: border-box;
+    padding: 10px;
+    text-align: center;
+}
+
+/* (D) ANIMATION - MOVE ITEMS FROM RIGHT TO LEFT */
+/* 4 ITEMS -400%, CHANGE THIS IF YOU ADD/REMOVE ITEMS */
+@keyframes tickerh {
+    0% {
+        transform: translate3d(100%, 0, 0);
+    }
+
+    100% {
+        transform: translate3d(-400%, 0, 0);
```

Comment:
The css implementation here is neat and compact, but I'm not sure it'll be flexible enough for us, e.g. it looks like it'll fit the whole animation into 30 seconds irrespective of whether there are 2 ticker items or 20...  I suspect we may need to invoke some javascript, which would have visibility of the number of items.